### DISCUSSION TITLE
Server Installation (Bug report)

### DISCUSSION THREAD

**Discussion by mangoose20026** [TriliumNext/Trilium#6235] [Q&A] [answered]
### Trilium Version

Latest Version

### What operating system are you using?

Windows

### What is your setup?

Server access only

### Operating System Version

Windows or Linux

### Description

Hi,
I'm the admin of the Trilium server installation https://github.com/zadam/trilium/wiki/Server-installation
Will i will be having only the admin access (readt/edit/delete) or everyone who access my webpage will have entire access

I should be the admin user and people who access my content should be a read-only user
Will all the user will get the search and advanced search ?

Please advise.

Thanks

### Error logs

No, errors but a query

**Comment by meichthys** [comment, marked as answer]
If the users have your password, they will have full access to your entire database. If you want to share your content, you should share your content using the share feature. That would only give the users read-only access using a public URL.

**Comment by mangoose20026** [comment]
If the user doesn't have my password. will they able to read and search without need of any credentials like a website???

**Comment by meichthys** [comment]
No, there is no search when accessing the content using a public link.

**Comment by meichthys** [comment]
If you don't need deep security, you can log the user into their own client without giving them the password. Then the client will stay logged in indefinitely and they will not know the password. You can then password protect all of the notes that you do not want them to access. They will be able to search and read any of the other notes that are not password protected.
